README.md

```diff
@@ -1,14 +1,19 @@
-#### To get started
+## To get started
 
+```sh
+yarn
+# or npm i
+yarn start
+# or npm start
```

Comment:
opinionated much? :D that's fine although bare in mind some people solving it aren't deep into nodejs dev so they might not be familiar with yarn. Npm is just available if you have node